src/status_im/common/home/actions/view.cljs

```diff
@@ -430,10 +433,16 @@
     [(group-details-entry chat-id)
      (when inside-chat?
        (if admin?
-         (manage-members-entry)
-         (add-members-entry)))
-     (when (and admin? inside-chat?) (edit-group-entry))
-     (when (and admin? inside-chat?) (group-privacy-entry))]))
+         (when config/show-not-implemented-features?
+           (manage-members-entry))
+         (when config/show-not-implemented-features?
+           (add-members-entry))))
+     (when (and admin? inside-chat?)
+       (when config/show-not-implemented-features?
+         (edit-group-entry)))
```

Comment:
Here I'm adding an extra layer of `when` expressions, and I did this because I thought it would be easier to replace this not-implemented feature with the implemented feature later on without worrying about the other states/booleans. That being said, I'm fine with flattening them too, I'm just hoping it will be simple to maintenance when someone else has to implement the features.

Reply:
100% agree, I think the nested approach is better
Easier to remove when feature is implemented and separates concerns between feature flags and component logic.